test-data/check-modules.yml

```diff
@@ -0,0 +1,17 @@
+# Type checker test cases dealing with modules and imports.
+
+- case: Access imported definitions
+  files:
+    - content: |
+        import m
+        import typing
+        m.f()           # E: Too few arguments for "f"
+        m.f(object())   # E: Argument 1 to "f" has incompatible type "object"; expected "A"
+        m.x = object()  # E: Incompatible types in assignment (expression has type "object", variable has type "A")
+        m.f(m.A())
+        m.x = m.A()
+    - path: m.py
+      content: |
```

Comment:
@ilevkivskyi I'm afraid not. `path` and `content` become the keys of the resulting dict: `{'path': 'm.py', 'content': '...'}`. I've made the file path `main.py` default so you don't have to type `path` for every test. But when you need a custom path, you have to provide it as an attribute.